azurerm/internal/services/cosmos/common/indexing_policy.go

```diff
@@ -0,0 +1,122 @@
+package common
```

Comment:
Added these to common as the Gremlin Graph resource could migrate over to these in the future